db/version_set.cc

```diff
@@ -122,7 +121,7 @@ class FilePicker {
     }
   }
 
-  int GetCurrentLevel() { return returned_file_level_; }
+  int curr_level() const { return curr_level_; }
```

Comment:
The function name should still be GetCurrentLevel following the convention of other function names in this class, isn't it?